src/interfaces/cloud.rs

```diff
@@ -0,0 +1,410 @@
+use std::{
+    collections::{HashMap, HashSet},
+    fmt::Display,
+    str::FromStr,
+};
+
+use cosmian_crypto_core::{bytes_ser_de::Serializable, CsRng};
+use rand::SeedableRng;
```

Comment:
Please, use `cosmian_crypto_core::reexport::rand_core::SeedableRng`. It would be great to be able to get rid of the `rand` dependency.

Reply:
Done in f00c141